---
int64 shoe_id
int64 goal_id
